File: src/main/java/com/p2p/lending/service/TradeService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Trade;


public interface TradeService {
	Map<String, Object> selecttd(String currpge, Map<String, Object> findmap);
	boolean addDate(Trade td);
	List<Trade> selectMoney(Integer uid);
}

File: src/main/java/com/p2p/lending/service/TradeService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Trade
import org.springframework.stereotype.Service

@Service
interface TradeService {
    fun selecttd(currpage: String, findmap: Map<String, Any>): Map<String, Any>
    fun addDate(td: Trade): Boolean
    fun selectMoney(uid: Int?): List<Trade>
}
